wallet/src/main/scala/org/bitcoins/wallet/DLCWallet.scala

```diff
@@ -252,6 +269,13 @@ abstract class DLCWallet extends Wallet {
       dlcPubKeys =
         DLCPublicKeys.fromExtPubKeyAndIndex(account.xpub, dlc.keyIndex, network)
 
+      fundingPrivKey =
+        extPrivKey
+          .deriveChildPrivKey(BIP32Path.fromString(s"m/0/${dlc.keyIndex}"))
```

Comment:
This path kinda seems like it deserves its own `def`, perhaps not even in this file?